tests/pipelines/test_sklearn_classifier.py

```diff
@@ -0,0 +1,124 @@
+# flake8: noqa
+from functools import partial
+import os
+
+from pytest import mark
+import torch
+from torch.optim import Adam
+from torch.utils.data import DataLoader
+
+from catalyst import data, dl
+from catalyst.contrib import datasets, models, nn
+from catalyst.data.transforms import Compose, Normalize, ToTensor
+from catalyst.settings import SETTINGS
+
+if SETTINGS.ml_required:
+    from sklearn.ensemble import RandomForestClassifier
+    from sklearn.linear_model import LogisticRegression
+    from sklearn.neighbors import KNeighborsClassifier
+    from sklearn.svm import SVC
+
+TRAIN_EPOCH = 30
+LR = 0.01
+RANDOM_STATE = 42
+
+
+def train_experiment(device, engine=None):
+    from catalyst import utils
+
+    utils.set_global_seed(RANDOM_STATE)
+    # 1. train, valid and test loaders
+    transforms = Compose([ToTensor(), Normalize((0.1307,), (0.3081,))])
+
+    train_dataset = datasets.MnistMLDataset(
+        split=11, root=os.getcwd(), download=True, transform=transforms
+    )
```

Comment:
could you use just a simple `MnistDataset`? why do you need an ML-based one?

Reply:
Done, let see what happened with the test.